tests/gnrc_ipv6_ext_frag/Makefile

```diff
@@ -51,3 +50,8 @@ ethos:
 	$(Q)env -u CC -u CFLAGS make -C $(RIOTTOOLS)/ethos
 
 include $(RIOTBASE)/Makefile.include
+
+# Set the pool size for limit objects if not being set by Kconfig
+ifndef CONFIG_GNRC_IPV6_EXT_FRAG_LIMITS_POOL_SIZE
+  CFLAGS += -DCONFIG_GNRC_IPV6_EXT_FRAG_LIMITS_POOL_SIZE=3
+endif
```

Comment:
Shouldn't there be a way to enforce this for the value by the application or have a minimum required? If I'm test-trolling and set this value in `make menuconfig` to a lower value than 3 the tests fail (naturally because now the full datagram can't be received).

Reply:
If this is an application-specific constraint for this parameter, then we can place a Kconfig file in the application folder with:
```
config GNRC_IPV6_EXT_FRAG_LIMITS_POOL_SIZE
   range <min> <max>
```
Is this your question?

Reply:
> If this is an application-specific constraint for this parameter, then we can place a Kconfig file in the application folder with:
> 
> ```
> config GNRC_IPV6_EXT_FRAG_LIMITS_POOL_SIZE
>    range <min> <max>
> ```
> 
> Is this your question?

Yes, but I'm not satisfied with your answer. Is there also something like

```
config GNRC_IPV6_EXT_FRAG_LIMITS_POOL_SIZE
   minimum <min>
```

?

Reply:
Also: wouldn't that force `kconfiglib` to be downloaded for this application, even if not required?

Reply:
> Yes, but I'm not satisfied with your answer. Is there also something like
> 
> ```
> config GNRC_IPV6_EXT_FRAG_LIMITS_POOL_SIZE
>    minimum <min>
> ```
> 
> ?

No, there is no `minimum` attribute.

> Also: wouldn't that force `kconfiglib` to be downloaded for this application, even if not required?

Yes, this will force Kconfig to run, which means that `kconfiglib` is downloaded.

We can also perform the check in the Makefile, and through an `$(error ...)` if is less than 3. But I think that giving the user the range in `menuconfig` is much more intuitive.

Reply:
From reviewing the code, the configurations look like intended. I can give it another test, but first: @miri64 is there something else that blocks?

Reply:
I'd rather have it, that it stays enforced, as it was before :-/.

Reply:
I added an `app.config` file for the default value, and modified the `SHOULD_RUN_KCONFIG` variable to be overridable from the application.

Reply:
(this was a comment I made yesterday but accidentally got stuck in review feature limbo ...)